src/PointwiseFunctions/AnalyticSolutions/GrMhd/BondiMichelMHD.hpp

```diff
@@ -0,0 +1,296 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <limits>
+#include <pup.h>
+
+#include "DataStructures/Tensor/TypeAliases.hpp"
+#include "Options/Options.hpp"
+#include "PointwiseFunctions/AnalyticSolutions/GeneralRelativity/KerrSchild.hpp"
+#include "PointwiseFunctions/Hydro/EquationsOfState/EquationOfState.hpp"
+#include "PointwiseFunctions/Hydro/Tags.hpp"  // IWYU pragma: keep
+#include "Utilities/MakeArray.hpp"            // IWYU pragma: keep
+#include "Utilities/TMPL.hpp"
+#include "Utilities/TaggedTuple.hpp"
+
+namespace grmhd {
+namespace Solutions {
+
+/*!
+ * \brief Bondi-Michel accretion with superposed magnetic field in Schwarzschild
+ * spacetime.
+ *
+ * Modulo the magnetic field, an analytic solution to the 3-D GrMhd system.
+ * The user specifies the sonic radius \f$r_c\f$ and sonic rest mass density
+ * \f$\rho_c\f$, which are the radius and rest mass density at the sonic point,
+ * the radius at which the fluid's Eulerian velocity as seen by a distant
+ * observer overtakes the local sound speed \f$c_{s,c}^2\f$. With a specified
+ * polytropic exponent \f$\Gamma\f$, these quantities can be related to the
+ * sound speed at infinity \f$c_{s,\inf}^2\f$ using the following relations:
+ *
+ * \f{align*}
+ * c_{s,c}^2 &= \frac{1}{2r_c - 3} \\
```

Comment:
You should mention what coordinates `r_c` and `r` are in. I'm a bit concerned the documentation is still not in one coordinate system.